lib/github/ldap.rb

```diff
@@ -68,6 +68,9 @@ class Ldap
     #
     def initialize(options = {})
       @uid = options[:uid] || "sAMAccountName"
+      @instrumentation_service = options[:instrumentation_service]
+      @admin_user = options[:admin_user]
+      @admin_password = options[:admin_password]
```

Comment:
I don't think the admin auth attributes are AD-specific, at least they weren't [being used that way before](https://github.com/github/github-ldap/pull/94/files#diff-efa5b2f99f979678d1cf84d090d3dfbdR82).

Also, turns out that instance variable was already set at the bottom of the initializer. I've taken this assignment out.